enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -59,20 +59,38 @@ func NewKey(task *repb.ExecutionTask, configurationHash, runnerID string) (*fcpb
 	}, nil
 }
 
-// manifestKey returns the key for the snapshot manifest
+// localManifestKey returns the key for the local snapshot manifest.
 //
 // Because we always want runners to use the newest manifest/snapshot, this
 // doesn't actually create a digest of the manifest contents. It just takes a
 // hash of shared properties, for which the snapshot should be shared
-func manifestKey(ctx context.Context, env environment.Env, s *fcpb.SnapshotKey) (*repb.Digest, error) {
+func localManifestKey(ctx context.Context, env environment.Env, s *fcpb.SnapshotKey) (*repb.Digest, error) {
 	gid, err := groupID(ctx, env)
```

Comment:
nit: Maybe add a comment explaining that we don't need this for the remote key, bc the cache handles group partitioning. In case someone wonders why these are different and tries to consolidate

Reply:
Done